Classes/ComArihiroTitestfairyModule.m

```diff
@@ -48,5 +48,64 @@ -(void)begin:(id)apitoken
     return;
 }
 
+-(void)pushFeedbackController:(id)args
+{
+    [TestFairy pushFeedbackController];
```

Comment:
I found a crash on this line.

Logs is here.

```
[ERROR] The application has crashed with an uncaught exception 'NSInternalInconsistencyException'.
[ERROR] Reason:
[ERROR] Only run on the main thread!
...
```

I think that it needs `ENSURE_UI_THREAD_0_ARGS` macro.
I tried to add the macro, then fixed it.